services/AUTOMATIC1111/Dockerfile

```diff
@@ -55,6 +55,11 @@ RUN pip install --prefer-binary --no-cache-dir -r ${ROOT}/repositories/CodeForme
 # TODO: move to top
 RUN apt-get install jq moreutils -y
 
+COPY . /docker
+RUN chmod +x /docker/*.sh
+
+ARG DEEPDANBOORU="0"
+RUN [[ ${DEEPDANBOORU:-"0"} == "0" ]] && : || /docker/optional-deepdanbooru.sh
```

Comment:
one final change: the script is nothing but a pip install, can you just put the pip install directly here?

then you would not need the copy and the chmod, also the change below won't be necessary